### ISSUE TITLE
Cameras

### ISSUE THREAD

**Issue by erictuvesson** [studio-nine/Nine.Graphics#5]
How should we handle cameras? Adding a Matrix parameter to _IRenderer<T>.Draw(...)_ ?

And OpenTK doesn't support _System.Numerics.Vector's_ for _GL.UniformMatrix4(...)_ . So we either have to convert the matrices or use unsafe code [like OpenTK](https://github.com/ChillyFlashER/opentk/blob/develop/Source/OpenTK/Graphics/OpenGL/GLHelper.cs#L664).

**Comment by yufeih** [comment]
> Adding a Matrix parameter to IRenderer.Draw(...) ?

Yeah, but we'll need **Matrix3x2** for 2D cameras and two **Matrix4x4** (view and projection) for 3D cameras. This cannot be added to `IRenderer<T>`, so we'll first remove the `IRenderer<T>` interface and add the parameters to `SpriteRenderer.Draw` and `ModelRenderer.Draw` methods.

> So we either have to convert the matrices or use unsafe code

**WP8/Silverlight** prevents the use of unsafe code. Now that our target is **corefx/Windows 10**, unsafe code can be used for performance sensitive code path.

**Comment by yufeih** [comment]
OpenTK is also discussing using `System.Numerics.Vectors` https://github.com/opentk/opentk/issues/245 , so the problem will go away when everything works on top of the same vector library.
